### PR TITLE
Api logging

### PR DESCRIPTION
**Pull request by MBishop17**
grammar and style pass for #776 (jason's PR closed in favor of this one).

### CODE REVIEW

**Comment by deniseschannon** [comment]
@MBishop17 i don't think we need to do the folder name change. we can just change the title and that should be fine.

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md [hunk 1]
```diff
@@ -1,63 +1,85 @@
 ---
-title: API Auditing
+title: API Audit Log
 weight: 10000
+aliases:
+- /rancher/v2.x/en/installation/api-auditing
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: let's revert back to what it was called in the directory so we don't have to keep up the aliases

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md [hunk 2]
```diff
@@ -1,63 +1,85 @@
...
 ---
 
-Rancher ships with API Auditing to record the sequence of system events initiated by individual users. You can know what happened, when it happened, who initiated it, and what cluster it affected. API auditing records all requests and responses to and from the Rancher API, which includes use of the Rancher UI and any other use of the Rancher API through programmatic use.
+You can enable the API audit log to record the sequence of system events initiated by individual users. You can know what happened, when it happened, who initiated it, and what cluster it affected. Then API audit log records all requests and responses to and from the Rancher API, which includes use of the Rancher UI and any other use of the Rancher API through programmatic use.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: Should the second sentence be ```The API audit log records all requests and responses```

Comment (aiwantaozi) [reply content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: The API audit log records all requests to the Rancher API and responses from it?

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md [hunk 3]
```diff
@@ -1,63 +1,85 @@
...
-
 <br/>
-#### Audit Level Logging
+
+### Audit Log Levels
 
 The following table displays what parts of API transactions are logged for each [`AUDIT_LEVEL`](#audit-level) setting.
 
-| `AUDIT_LEVEL` Setting | Request Header     | Request Body | Response Header     | Response Header     |
+| `AUDIT_LEVEL` Setting | Request Header     | Request Body | Response Header     | Response Body       |
```
Comment (aiwantaozi) [inline content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: Request Header -> Request Metadata
Response Header -> Response Metadata

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md [hunk 4]
```diff
@@ -1,63 +1,85 @@
...
 | `2`                   | ✓                  | ✓            |                     |                     |
 | `3`                   | ✓                  | ✓            | ✓                   | ✓                   |
 
-## Enabling API Auditing
+## Viewing API Audit Logs
 
-To enable API auditing, stop the Docker container that's running Rancher, and then restart it using the following command. This command includes parameters that turns on API auditing. For more information about usage for each switch related to API auditing, see [API Auditing Usage](#api-auditing-usage).
+### Single Node Install
 
+Share the `AUDIT_LOG_PATH` directory (Default: `/var/log/auditlog`) with the host system. The log can be parsed by standard CLI tools or forwarded on to a log collection system.
```
Comment (aiwantaozi) [inline content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: The log can be parsed by standard CLI tools or forwarded to a log collection system by log collection tools like Fluentd/Filebeat/Logstash etc.

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md [hunk 5]
```diff
@@ -1,63 +1,85 @@
...
-  -e AUDIT_LOG_PATH=/var/log/auditlog/rancher-api-audit.log \
-  -e AUDIT_LOG_MAXAGE=20 \
-  -e AUDIT_LOG_MAXBACKUP=20 \
-  -e AUDIT_LOG_MAXSIZE=100 \
-  rancher/rancher:latest
-```
+### HA Install
 
-## Viewing API Audit Logs
+Enabling the API Audit Log with the Helm chart install will create a `rancher-audit-log` sidecar container in the Rancher pod. This container will stream the log to standard output (stdout). You can view the log as you would any container log.
```
Comment (aiwantaozi) [inline content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: Do we have image for this rancher-audit-log sidecar container?

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md (hunk 1 above)
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: @deniseschannon , I'm unclear what you're asking for. Do you want me to change the title back to what it was, or move the file back to the installation folder?

Comment (deniseschannon) [reply content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: ignore my comment

File: content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md (hunk 5 above)
Comment (deniseschannon) [reply content/rancher/v2.x/en/admin-settings/api-audit-log/_index.md]: No, there is no defined image.

**Comment by aiwantaozi** [comment]
LGTM

**Pull request merged**